oeps/best-practices/oep-0067-bp-tools-and-technology.rst

```diff
@@ -215,6 +223,14 @@ Frontend Technology Selection
 
    You can find out more about Sass in the official `Sass documentation`_.
 
+.. _use-css-variables:
+
+#. **CSS variables should be used to support runtime theming**
+
+   **Rationale**: SCSS variables are evaluated by the SASS preprocessor, and cannot be
+      changed afterwards. CSS variables are evaluated by the browser and
+      therefore can be changed at runtime, enabling dynamic theming support.
```

Comment:
It looks like moving the ref prevents the number reset issue

```suggestion
#. **CSS variables should be used to support runtime theming**

   .. _use-css-variables:

   **Rationale**: SCSS variables are evaluated by the SASS preprocessor, and cannot be
      changed afterwards. CSS variables are evaluated by the browser and
      therefore can be changed at runtime, enabling dynamic theming support.
```

it's not an *ideal* solution as it causes the page to scroll just a little bit too far

![image](https://github.com/user-attachments/assets/12a4a693-06f1-4ac8-9545-53268573f2ae)

but it feels "good enough" imo.